core/src/main/java/com/datastax/oss/driver/api/core/type/codec/CodecNotFoundException.java

```diff
@@ -62,4 +63,11 @@ public DataType getCqlType() {
   public GenericType<?> getJavaType() {
     return javaType;
   }
+
+  @NonNull
+  @Override
+  public DriverException copy() {
+    return new CodecNotFoundException(
+        getMessage(), getCause(), getCqlType(), getJavaType());
```

Comment:
The build is failing due to wrong format:

```suggestion
return new CodecNotFoundException(getMessage(), getCause(), getCqlType(), getJavaType());
```